packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -89,7 +89,8 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
                   <div className={this._classNames.chartTitle}>
                     {points!.chartTitle && (
                       <div
-                        className={this._classNames.chartDataText}
+                        className={this._classNames.chartDataTextLeft}
+                        title={points!.chartTitle}
```

Comment:
Should we be using the alt attribute.
https://www.a11yproject.com/posts/title-attributes/

Reply:
The alt attribute can only be used with img. To use it with div, we will have to set the role attribute to img. Also, the alt text doesn't get displayed in a tooltip.